test/vaadin-combo-box-dropdown.html

```diff
@@ -55,6 +68,57 @@
         });
       });
     });
+
+    describe('frontmost', () => {
+      var dialog;
+      beforeEach(() => {
+        dialog = fixture('in-modeless-dialog');
+        dialog.opened = true;
+      });
+
+      function touchstart(node) {
+        const event = new CustomEvent('touchstart', {bubbles: true, cancelable: true});
+    
+        const nodeRect = node.getBoundingClientRect();
+        const clientX = nodeRect.left;
+        const clientY = nodeRect.top;
+
+        event.touches = event.changedTouches = event.targetTouches = [{clientX, clientY}];
+        node.dispatchEvent(event);
+      }
+
+      it('should not end up behind the dialog overlay on mousedown', (done) => {
+        const dialogOverlay = dialog.$.overlay;
+        const cbBox = dialog.$.overlay.querySelector('vaadin-combo-box');
+        cbBox.items = ['1', '2'];
+
+        cbBox.open();
+        const cbBoxDropdownWrapper = cbBox.$.overlay;
+        Polymer.RenderStatus.afterNextRender(cbBox, () => {
```

Comment:
Is this necessary?

Reply:
checked, and it is not necessary. I removed it.